src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -407,9 +428,46 @@ private-f net.minecraft.world.level.storage.loot.LootPool rolls # rolls
 private-f net.minecraft.world.level.storage.loot.LootPool bonusRolls # bonusRolls
 public net.minecraft.server.network.ServerConfigurationPacketListenerImpl finishCurrentTask(Lnet/minecraft/server/network/ConfigurationTask$Type;)V
 public com.mojang.blaze3d.vertex.VertexFormatElement$Usage$SetupState
-public net.minecraft.world.level.block.LiquidBlock fluid
-public net.minecraft.world.item.BucketItem content
-public net.minecraft.world.item.ItemStack addToTooltip(Lnet/minecraft/core/component/DataComponentType;Lnet/minecraft/world/item/Item$TooltipContext;Ljava/util/function/Consumer;Lnet/minecraft/world/item/TooltipFlag;)V
-public net.minecraft.world.item.SwordItem createToolProperties()Lnet/minecraft/world/item/component/Tool;
-public net.minecraft.world.level.block.CropBlock hasSufficientLight(Lnet/minecraft/world/level/LevelReader;Lnet/minecraft/core/BlockPos;)Z # hasSufficientLight
-public net.minecraft.world.item.crafting.ShapedRecipe pattern # pattern
\ No newline at end of file
+
+# Below you will find particle constructor ATs
```

Comment:
I was thinking about this, during the last few days:
I think we might want to introduce a <init>:all mode where all constructors in the class and all of its inheriting types are forced to the visibility set or higher.

That way we will ensure that we won't miss a new particle in the future?

Reply:
That's beyond my skillset as I wouldn't know where to begin or how to add that feature to ATs. Could it be done after this PR? Like add the feature, then go back and clean up our ATs? 

if it is setup to ignore abstract classes, we can use it to clean up our Block constructor ATs too

Reply:
Yeah sure, just wanted to get your feedback on the idea.